### PR TITLE
Ayomiku #860qytw3k create detail page with hardcoded data

### PR DESCRIPTION
**Pull request by Ayomikuolatunji**

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -1,14 +1,272 @@
...
+    fontWeight: {
+      400: "400",
+      500: "500",
+      600: "600",
+      700: "700",
+    },
+    borderRadius: {
+      none: "0",
+      sm: "0.125rem",
+      DEFAULT: "0.25rem",
```
Comment (hhimanshu) [inline tailwind.config.js]: Why are there 2 `DEFAULT` defined?

File: tailwind.config.js [hunk 2]
```diff
@@ -1,14 +1,272 @@
...
+        css: {
+          fontSize: "14px",
+          lineHeight: "20px",
+          fontStyle: "normal",
+        },
+      },
+    },
+    extend: {
+      colors: {
+        primary: "#4C42E8",
```
Comment (hhimanshu) [inline tailwind.config.js:231]: How can we get material theme color references here, such as `primary95`, `neutral50`, etc?

File: gatsby-ssr.jsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+import { App } from "konsta/react";
+import "@fontsource/poppins";
+import "@fontsource/poppins/500.css";
+import "@fontsource/poppins/600.css";
+import "@fontsource/poppins/700.css";
+import "./src/styles/styles.css";
+
+export const wrapPageElement = ({ element }) => {
+  return (
+    <App theme="material" dark={false} className="font-sans">
```
Comment (hhimanshu) [inline gatsby-ssr.jsx]: What's the purpose of `className="font-sans"?`

File: gatsby-browser.jsx [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { App } from "konsta/react";
+import "@fontsource/poppins";
+import "@fontsource/poppins/500.css";
+import "@fontsource/poppins/600.css";
+import "@fontsource/poppins/700.css";
+import "./src/styles/styles.css";
+
+export const wrapPageElement = ({ element }) => {
```
Comment (hhimanshu) [inline gatsby-browser.jsx:9]: I saw exact same code in `gatsby-ssr`, why are there duplicates?

File: src/styles/styles.css [hunk 5]
```diff
@@ -1,13 +1,22 @@
-html {
-  overflow: auto;
-  font-family: "Poppins" !important;
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap");
```
Comment (hhimanshu) [inline src/styles/styles.css]: We should get fonts using fontSource. See https://fontsource.org/fonts/poppins/install for more details

File: src/pageTemplates/DetailsPage.tsx [hunk 6]
```diff
@@ -0,0 +1,236 @@
+import React from "react";
+// import { PageProps } from "gatsby";
```
Comment (hhimanshu) [inline src/pageTemplates/DetailsPage.tsx]: Shall we remove this file?
If you need code, you can keep this file handy elsewhere?

File: gatsby-browser.jsx (hunk 4 above)
Comment (Ayomikuolatunji) [reply gatsby-browser.jsx:9]: One is used to load the Konsta UI on server side while the other on the client side

File: gatsby-ssr.jsx (hunk 3 above)
Comment (Ayomikuolatunji) [reply gatsby-ssr.jsx]: I will remove

Comment (Ayomikuolatunji) [reply gatsby-ssr.jsx]: fixed

File: src/styles/styles.css (hunk 5 above)
Comment (Ayomikuolatunji) [reply src/styles/styles.css]: Yeah fixed

File: src/pageTemplates/DetailsPage.tsx (hunk 6 above)
Comment (Ayomikuolatunji) [reply src/pageTemplates/DetailsPage.tsx]: Yes I cleanup the code so we can start adding new code as per use of task.

File: src/styles/styles.css (hunk 5 above)
Comment (hhimanshu) [reply src/styles/styles.css]: I do not see entry in `package.json` to install the font from `npm`

File: tailwind.config.js (hunk 2 above)
Comment (Ayomikuolatunji) [reply tailwind.config.js:231]: we can

Comment (Ayomikuolatunji) [reply tailwind.config.js:231]: We can create a new task for it

**Review by hhimanshu — APPROVED** [review]

**Pull request merged**